lib/loaders/gltf-loader.ts

```diff
@@ -91,7 +91,7 @@ class WTGLTFLoader {
 
             // load material textures
             if (threeMat.map !== null) {
-                material.albedoTexture = new Texture(threeMat.map.image);
+                material.albedoMap = new Texture(threeMat.map.image);
                 material.baseColor = new THREE.Vector3(-1, -1, -1);
```

Comment:
We can remove this line.

Reply:
material.baseColor .... should be removed already